packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -692,6 +693,102 @@ const MeetingUtil = {
       EVENT_TRIGGERS.MEETING_INTERPRETATION_UPDATE
     );
   },
+
+  /**
+   * Returns a CA-recognized error payload for the specified raw error message/reason.
+   *
+   * New errors can be added to this function for handling in the future
+   *
+   * @param {String} reason the raw error message
+   * @returns {Array<object>} an array of payload objects
+   */
+  getChangeMeetingFloorErrorPayload: (reason: string) => {
+    const errorPayload = {
+      errorDescription: reason,
+      name: 'locus.response',
+      shownToUser: false,
+    };
+    if (reason.includes(LOCAL_SHARE_ERRORS.UNDEFINED)) {
+      return [
+        {
+          ...errorPayload,
+          fatal: true,
+          category: 'signaling',
+          errorCode: 1100,
+        },
+      ];
+    }
+    if (reason.includes(LOCAL_SHARE_ERRORS.DEVICE_NOT_JOINED)) {
+      return [
+        {
+          ...errorPayload,
+          fatal: true,
+          category: 'expected',
```

Comment:
why is it expected?

Reply:
I'm assuming this error occurs when a user attempts to share before their device connects? I'm assuming it's expected that this could happen?

Reply:
the web app should make sure that you can only share after you've joined, so this is not expected - if this happens it's a bug in the web app (or sdk) that needs fixing